py/nightwatch/webpages/amp.py

```diff
@@ -36,8 +36,11 @@ def write_amp_html(outfile, data, header):
         program = header['PROGRAM'].rstrip()
     exptime = header['EXPTIME']
 
-    env = jinja2.Environment(
-        loader=jinja2.PackageLoader('nightwatch.webpages', 'templates')
+      env = jinja2.Environment(
```

Comment:
post-facto: @ana-lyons this indentation looks wrong, off by 2 spaces.  Please check and fix directly in master if needed.

Reply:
Fixed!